### ISSUE TITLE
Error: "cp: preserving permissions for ‘....’: Permission denied"

### ISSUE THREAD

**Issue by Sfinx** [rfjakob/gocryptfs#491]
1.8.0 gocryptfs stock static binary at Ubuntu 18.04 with 5.7.10-050710-lowlatency kernel. To reproduce:

```
rus@Shiva:~$ md crypt plain
rus@Shiva:~$ gocryptfs -init crypt
Choose a password for protecting your files.
Password: 
Repeat: 

Your master key is:
  ...
If the gocryptfs.conf file becomes corrupted or you ever forget your password,
there is only one hope for recovery: The master key. Print it to a piece of
paper and store it in a drawer. This message is only printed once.
The gocryptfs filesystem has been created successfully.
You can now mount it using: gocryptfs crypt MOUNTPOINT
rus@Shiva:~$ gocryptfs crypt plain
Password: 
Decrypting master key
Filesystem mounted and ready.
rus@Shiva:~$ cd plain/
rus@Shiva:~/plain$ git clone https://github.com/rfjakob/gocryptfs
Cloning into 'gocryptfs'...
remote: Enumerating objects: 374, done.
remote: Counting objects: 100% (374/374), done.
remote: Compressing objects: 100% (187/187), done.
remote: Total 9790 (delta 254), reused 286 (delta 187), pack-reused 9416
Receiving objects: 100% (9790/9790), 5.05 MiB | 3.47 MiB/s, done.
Resolving deltas: 100% (6774/6774), done.
rus@Shiva:~/plain$ cp -a gocryptfs 1
cp: preserving permissions for ‘1/.git/objects/pack/pack-269ca257621d070fd963361ce7b55b3818158954.pack’: Permission denied
cp: preserving permissions for ‘1/.git/objects/pack/pack-269ca257621d070fd963361ce7b55b3818158954.idx’: Permission denied
```

**Comment by Sfinx** [comment]
strace shows that file do not have the ACL, but cp tries to set them. is this cp broken or gocryptfs ? ;)

```
fchmod(4, 0400)                         = 0
fgetxattr(3, "system.posix_acl_access", 0x7ffdbbd16860, 132) = -1 ENODATA (No data available)
fstat(3, {st_mode=S_IFREG|0444, st_size=2430372, ...}) = 0
fsetxattr(4, "system.posix_acl_access", "\2\0\0\0\1\0\4\0\377\377\377\377\4\0\4\0\377\377\377\377 \0\4\0\377\377\377\377", 28, 0) = -1 EACCES (Permission denied)
fsetxattr(4, "system.posix_acl_access", "\2\0\0\0\1\0\4\0\377\377\377\377\4\0\4\0\377\377\377\377 \0\4\0\377\377\377\377", 28, 0) = -1 EACCES (Permission denied)
fchmod(4, 0100444)                      = 0
```

**Comment by rfjakob** [comment]
Hmm, what filesystem is this on? I don't understand where the EACCESS comes from.

Does this work in a normal directory, and also in a gocryptfs mount?

   touch file1
   setfattr -n user.foo -v bar file1

**Comment by Sfinx** [comment]
It is ext4 fs from Ubuntu 18.04. This work as at normal directory as in gocryptfs mount.

**Comment by rfjakob** [comment]
I can reproduce this inside https://app.vagrantup.com/generic/boxes/ubuntu1804

**Comment by rfjakob** [comment]
Works ok with v2.0-beta3: https://github.com/rfjakob/gocryptfs/releases/tag/v2.0-beta3

This was probably fixed by disabling ACL encryption in v2.0.
